scripts/mkcloud

```diff
@@ -167,6 +167,11 @@ host_mtu=$(determine_mtu)
 iscloudver 7plus && [[ $arch == x86_64 ]] && \
     [[ -n $want_efi ]] && firmware_type="uefi"
 
+# TODO(colleen) Fix cloudsource check once there are new MUs for cloud7/8
+iscloudver 7plus && [[ $cloudsource =~ develcloud ]] && {
+    export ssl_trusted=1
```

Comment:
maybe want_ssl_trusted to be in the normal "want_XXX" pattern of variables. we could also add it to the documentation then in lib/qa_crowbarsetup-help.sh  (want_all_ssl is missing there as well)?